docs/notebooks/tutorial.ipynb

```diff
@@ -449,9 +518,9 @@
ekIthBgyRKYO7f8cWYKk/Tly/MkfdOmyn1mT3HmmXl9yZKOv98kXZIk\nSZKqrL1J+oEH5m3/+lfxM9mu6wBbbw077VS++JrKkvQXXoB//ANGjXK6eybGvP7SSx1/v0m6JEmS\nJFVZW0l6dpzaPffA3/6W6s88U/zM009XJrZSsiT9vPPya6e7J4Xr0Dsz3d3d3SVJkiSpyto7kj5i\nBEyenJLiSy4pfmbmzMrF11RdXUrSs7g2b05tTndPO+4PH55mPfzylx1/vyPpkiRJklRlbSXpP/85\nfOITqV5XB5Mm5fcefRT++teUvHeXbCR9q63S9eGHO5IOsHAhfPObsGoVPPZY5/owSZckSZKkKmsr\nST/gALjiivx64sS8Pn48HHZY5WIrJUvSd9stXb/tbY6kQ/E+AXvtldeHDGl/HybpkiRJklRlbSXp\nTY0aldfHjy9/PG3JkvSGBth229TmSHrxpnG//31ev+aa9vdhki5JkiRJVZYdZ9bRJP3ooysTT1sK\nk/RsUzuTdFi9OpUjRsAWW+Tto0e3vw+TdEmSJEmqssWLU7nllu17PkvSOzKNupxKJelOd4c1a1I5\nb14qQ0jl0KHt78MkXZIkSZKq7Jln0sjr2LHtez4bma2lJN2R9DxJz5LybPr78OHt78MkXZIkSZKq\n6JVXYPZs2GOPfOS1LbU6kt40Sb/55vSd+soI+5o16ftmP5NMXV37+/CcdEmSJEmqkjlz8h3Szz67\n/e8bMSKV1UrSFy6EJUvS8W/ZHwzuvTftbr5kCYwZk9p+/ONUrlpVvNldb7VmTRpFz/7Ycu+9cNdd\nHevDkXRJkiRJqpJHHsnr22/f/vcNHJjKwYPLG097zZ2byscey0eN//a3VC5YkD/XrzHjXLmy+2Kr\nptWri9efH3ssXHppx/owSZckSZKkKsk2jIN89Lk9siS9f5XmRmefD82ndhfG1NeS9DVrYNiwrvVh\nki5JkiRJVbJqVV7vyNT1LEnuyFrnclq/Pq83TdL33z8/9z2THU3W22XT3bvCJF2SJEmSqqQwee3I\n1PXsPPVaSNJLxTB/fvFzhc/3ZibpkiRJktSDdTZJz0bS+1UpozvssLy+bl0qs03iAJYuTWU2om6S\n3n4m6ZIkSZJUJYVJekemu1d7JP3rX4drr0317Hi1o4/O72ffq7cm6bfcAj/7WfP2ciTpHsEmSZIk\nSVVSmKRniXdHVCtJr6uDXXZJ9RhTWbiZ3LHHwq235sl50zXqPd27353KQw+FHXfM21evduM4SZIk\nSeqxVq+GQw6Biy6CI49s//uyBPm44yoTV3tk0/OzJH38+OL7t9/ee0fSM5/7XCqvuCIdp1eO3d0d\nSZckSZKkKlm9GnbdFb797Y69b+ed8+S4Wpom6YUj6QBr1/b+JP2Xv4Tzz4errkrXU6a4Jl2SJEmS\neqRNm+DZZ2HbbasdSec0TdIBXnwxr69b1zd2d88SdOilG8eFED4aQng8hFDf+JoRQnhrG+85JoQw\nK4SwLoQwJ4Tw/u6KV5IkSZI64+GHYdEiOPXUakfSOdnO8jvtlLdNmpTX167Nk/MPfAA2b+6+2Cqt\npSntixfD6NFd67vmknRgAfA5YDIwBfgTcFsIYe9SD4cQdgTuBP4I7A9cDvw4hPDm7glXkiRJkjpu\n7txUHnhgdePorJ13hh/8AL7zndL3161LI8uQZg3cdlv3xVZJTz5ZvOFfoddeg//4j671X3NJeozx\njhjjXTHGuTHG52OMFwMrgUNaeMt5wNwY42djjP+MMV4F/AqY1l0xS5IkSVJH1denKeMdOR+9loQA\nH/94y6PKa9akZPaYY9L1aad1W2gVdf/9rd8//PCu9V9zSXqhEEJdCOGdwCDgLy08djhwb5O2exrb\nJUmSJKkmrVgBo0ZVO4rKefjhNII+ZUq1Iymvtv6o0iuPYAsh7BtCWAWsA34EnBVjfL6Fx8cBi5q0\nLQJGhhAGVTBMSZIkSeq0L385rUnv7bq6kVqtWboUttoKJk/O1+B/7Wvl679Wj2B7FtgPGAWcCfws\nhHBMjHFWdcOSJEmSpK7rTZuotWW33aodQXktWQJbbw0zZ8Idd8App6T2t78d9tmn6/3XZJIeY9wA\nvNB4+WgI4WDgo8CHSjz+KjC+Sds44PUYY0NLnzFt2jRGNZlbMnXqVKZOndrpuCVJkiSpPX73u1Re\ne2114+gOO+4IH/sYzJiRXk8/DR8qldn1EEuXpiQd8uPz9twTvvSlVJ8+fTrTp08vek99fX27+6/J\nJL2EOlqemv9X4MQmbccDM1rr8Hvf+x6TJ08uQ2iSJEmS1H6PPJKPvu6/f3Vj6Q5Dh6a19/X1cMQR\nqa2nJ+ljxqT6AQfAc8/Brrvm90sN/s6aNYsp7VycX3Nr0kMIl4QQjgwh7NC4Nv0S4Cjg5oL7Pyl4\ny/8CO4UQvh1C2COE8DHSFPnvdX/0kiRJktS6l17K64Xnivc2731vKmOEkSPh9derG09rnnwSZs9u\n37PZdPdMYYJeDrU4kj4GuBGYANQDjwNviTH+qfH+eGC77OEY47wQwkmkpPwC0jnr58QY/9CtUUuS\nJElSO2zYkNcLk73e4vTT0zFkBx8MP/0pbL997Sfp++2XyhhT+ZOfpPj32qv5s4XT3Suh5pL0GOO5\nbdz/QIm2+wHnrkuSJEmqeevX5/UQqhdHpfz613k9S3pHjSr+40QtabqJX4xw9tkwcWLxrIdM05H0\ncqu56e6SJEmS1JstW1btCLrfyJHVjqBlc+fm9cMPh+XLU33AgObPvv46rFoF22xTuXhM0iVJkiSp\nG738cionTqxuHN2plpP055/P6w8/DPPmpfr4pmeIAfPnp7KSewmYpEuSJElSN8oSvW99q7pxdKcR\nI6odQctmNDkX7OqrU9lakr799pWLxyRdkiRJkrrR3LlpzfN73lPtSLrPo
```

Comment:
In the image, it looks like AAPL's stock price has discontinuities from splits (I notice a jump in 2005). Is that expected?